widgets/refinement-list.js

```diff
@@ -51,17 +57,30 @@ function refinementList({
     throw new Error(usage);
   }
 
+  if (templates !== defaultTemplates) {
+    templates = Object.assign({}, defaultTemplates, templates);
```

Comment:
If you did not pass any `templates: {}` then we do not need to merge anything. is that ok?